app/src/main/AndroidManifest.xml

```diff
@@ -32,7 +32,7 @@
             android:exported="false" />
         <activity
             android:name=".activities.MainActivity"
-            android:exported="false">
+            android:exported="true">
```

Comment:
Here even though it's practical for testing, I don't think that we want our main activity app to be able to be launched from any component of other applications.

I wound probably change it to `false` to put it on the `master` branch.

Reply:
Yes you're right!